monitoring/localRun.ts

```diff
@@ -1,6 +1,44 @@
+import { prompt } from 'inquirer';
+import type { CustomScheduleEventContent } from './src';
+
+type LocalRunCLIUserInput = {
+	stage: string;
+	jurisdictions: string[];
+};
+
 async function main() {
 	const { handler } = await import('./src');
-	handler().catch(console.error);
+
+	await prompt([
+		{
+			type: 'list',
+			name: 'stage',
+			message: 'Which environment would you like to test?',
+			choices: ['code', 'prod'],
```

Comment:
If this is running locally we may also want to add the option to run this against the URLs DCR/frontend presents when run locally.